runtime/oti/j9nonbuilder.h

```diff
@@ -252,6 +252,10 @@
 #define J9DescriptionCpTypeShift 0x4
 #define J9DescriptionImmediate 0x1
 
+/* Rom CP Flag for constant dynamic with primitive type */
+#define J9DescriptionCpPrimitiveType 1 << 20
+#define J9DescriptionCpBsmIndexMask 0xFFFF
```

Comment:
Is this constant correct if the top bit is used for primitives?

Reply:
this is mask is suppose to get rid of the primitive bit.
`bsmIndex = (bsmIndexAndCpType >> J9DescriptionCpTypeShift) & J9DescriptionCpBsmIndexMask`
so we only get the U_16 value for the bsmIndex